GnuPlot.cs

```diff
@@ -423,7 +424,11 @@ public static void WriteData(double[] x, double[] y, double[] z, StreamWriter st
             {
                 if (i > 0 && x[i] != x[i - 1])
                     stream.WriteLine("");
-                stream.WriteLine(x[i] + " " + y[i] + " " + z[i]);
+
```

Comment:
string.Format accepts CultureInfo, no need to call ToString for every parameter
`string.Format(CultureInfo.InvariantCulture, "{0} {1} {2}", x[i], y[i], z[i]);`